views/product-listing.hbs

```diff
@@ -0,0 +1,39 @@
+<section class="product-listing top-padding">
+    <div class="wrapper">
+        <div class="row">  
+            <div class="product-listing__prod-filter col-xs-12 col-sm-3 col-md-3" id="page-nav">
+                <!-- [THE HAMBURGER] -->
+                <label for="product-hamburger">Fruits &amp; Vegetables</label>
+                <input type="checkbox" id="product-hamburger"/>
+                <ul class="product-listing__prod-categories">
+                    {{#each categories}}
+                        <li><a href="/product-listing/{{this.id}}">{{this.name}}</a></li>
+                    {{/each}}
+                </ul>
+            </div>
+           
+            <div class="product-listing__product-list col-xs-12 col-sm-9 col-md-9">
+                <div class="product-detail row">
+                    {{#each products}}
+                    <div class="col-xs-12 col-sm-6 col-md-3">
+                        <div class="product-listing__product-box row">
+                            <h3 class="col-xs-12">{{this.name}}</h3>
+                            <div class="col-xs-6 col-sm-6 col-md-12 product-listing__prod-image">
+                                <img src="{{this.imageURL}}" alt="{{this.name}}">
+                            </div>
+                            <div class="col-xs-6 col-sm-6 col-md-12 product-listing__prod-desc">
+                                <p>{{this.description}}</p>
+                                <div class="product-listing__prod-pricing">                                            
+                                    <a class="button-small" role="button" onclick="addProduct('{{this.id}}')">Buy Now
+                                        <span>MRP Rs.{{this.price}}</span>
+                                    </a>
```

Comment:
These labels should not be hard coded, use translations

Reply:
Done.